### PR TITLE
Fix a few minor CS issues

### PR DESCRIPTION
**Pull request by leofeyer**
| Q                | A
| -----------------| ---
| Fixed issues     | -
| Docs PR or issue | -

### CODE REVIEW

**Review by aschempp — APPROVED** [review]

**Review by fritzmg — APPROVED** [review]

File: core-bundle/src/Controller/AbstractController.php [hunk 1]
```diff
@@ -19,7 +19,7 @@
 
 abstract class AbstractController extends SymfonyAbstractController implements ServiceAnnotationInterface
 {
-    public static function getSubscribedServices()
+    public static function getSubscribedServices(): array
```
Comment (ausi) [inline core-bundle/src/Controller/AbstractController.php:22]: Isn’t this a BC break?

Comment (leofeyer) [reply core-bundle/src/Controller/AbstractController.php:22]: Hm, I guess you are right. If we added the return type, every child class has to add it as well. (Which is already the case for all our child classes.)

When did we add the abstract controller?

Comment (bytehead) [reply core-bundle/src/Controller/AbstractController.php:22]: In 4.9 (see https://github.com/contao/contao/pull/603).

Comment (leofeyer) [reply core-bundle/src/Controller/AbstractController.php:22]: We should have added the return type there. @contao/developers I assume we cannot add it as a bugfix in version 4.9.3, can we?

Comment (bytehead) [reply core-bundle/src/Controller/AbstractController.php:22]: I would consider this as BC break.

**Pull request closed without merging**